transforms/lmru_stakeholders.py

```diff
@@ -0,0 +1,20 @@
+{
+    'sheet': 'Structure',
+    'lets': {
+        'iri': 'vm:stakeholder/{row[Category / Stakeholder name].as_slug}',
+    },
+    'non_unique': ['vm:name', 'vm:description'],
+    'triples': [
+        ('{iri}', 'rdf:type', 'vm:Stakeholder'),
+        ('{iri}', 'vm:name', '{row[Category / Stakeholder name].as_json}@en'),
```

Comment:
`as_text` doesn't like cyrillic characters I guess?

Reply:
It's fine with non-ascii, but I want the quotes, as this ends up as a slightly dodgy (turtle language tagged literal)[https://www.w3.org/TR/turtle/#turtle-literals] analogue, so `"Designer"@en` (which is then quoted again in the json output).